packages/devtools_app/lib/src/screens/vm_developer/object_inspector_view_controller.dart

```diff
@@ -40,10 +28,30 @@ class ObjectInspectorViewController extends DisposableController
   ValueListenable<bool> get refreshing => _refreshing;
   final _refreshing = ValueNotifier<bool>(false);
 
+  ValueListenable<bool> get initialized => _initialized;
+  final _initialized = ValueNotifier<bool>(false);
+
   void init() {
-    programExplorerController
-      ..initialize()
-      ..initListeners();
+    if (!initialized.value) {
+      programExplorerController
+        ..initialize()
+        ..initListeners();
+      initListeners();
+      selectAndPushMainScript();
```

Comment:
why did we not have to call selectAndPushMainScript in init before this change?

Reply:
It had not beet added and the initial screen in the object viewport said 'No object selected', I thought it would be good to have the main script instead